src/main/webapp/app/shared/date-time-picker/date-time-picker.component.ts

```diff
@@ -82,13 +92,18 @@ export class FormDateTimePickerComponent implements ControlValueAccessor {
     }
 
     /**
-     *
-     * @param newValue
+     * updates the value with the passed newValue date. The date picked using the calendar is always valid.
+     * In case the calendar was opened and no date was selected, the value does not get updated
+     * @param newValue the date picked with the date picker or another object that is not date in case an invalid value
+     * is selected in the user input
      */
-    updateField(newValue: dayjs.Dayjs) {
-        this.value = newValue;
-        this._onChange(dayjs(this.value));
-        this.valueChanged();
+    updateField(newValue: any) {
+        if (this.isValidDate(newValue)) {
```

Comment:
I would also like to avoid any here, maybe we can use multiple types separated with `|`?